src/web_server.c

```diff
@@ -51,6 +51,39 @@ static void log_allocations(void)
 }
 #endif
 
+#ifndef HAVE_ACCEPT4
+int accept4(int sock, struct sockaddr *addr, socklen_t *addrlen, int flags) {
+	int fd = accept(sock, addr, addrlen);
```

Comment:
You are right.
I have a task to re-arrange the code so that it will be cleaner (web_client.* is misleading, we are going to need a libcurl or something for health monitoring, etc).

I'll do them all together when I find some spare time.